src/modules/page.js

```diff
@@ -108,19 +78,23 @@ class Page extends React.PureComponent {
 Page.ActionBar = PageActionBar;
 Page.Container = PageContainer;
 Page.Content = PageContent;
-Page.Details = PageDetails;
+Page.DataCards = PageDataCards;
+Page.DataGrid = PageDataGrid;
 Page.DataGroup = PageDataGroup;
 Page.DataGroups = PageDataGroups;
+Page.Details = PageDetailsWindow; // TODO: Deprecated. Alias name for Page.DetailsWindow. Remove in next major release.
+Page.DetailsWindow = PageDetailsWindow;
 Page.FiltersDrawer = PageFiltersDrawer;
 Page.FiltersRail = PageFiltersRail;
-Page.Grid = PageGrid;
-Page.Table = PageTable;
+Page.Grid = PageDataCards; // TODO: Deprecated. Alias name for Page.DataCard. Remove in next major release.
+Page.Table = PageDataGrid; // TODO: Deprecated. Alias name for Page.DataGrid. Remove in next major release.
```

Comment:
How widespread is the usage in HC?  Is it worth deprecating?  Or should we kill altogether and replace all the references in HC?

Reply:
Not to wide. I will fix in dev and let everyone know of a possible regression and what to do for their branches. In my eyes, super simple fix.

Reply:
But you'd rather leave the old aliases just in case?
Honestly, I'm pretty certain we're the only consumers ... I'd almost rather remove, let it fail hard until fixed.  Not the way we'd do things if it really were a shared library, and if you do want to do the deprecation and them remove in next major version, I am okay with that too.